airbyte-integrations/connectors/source-linkedin-leads/source_linkedin_leads/schemas/lead_form.json

```diff
@@ -0,0 +1,284 @@
+                                    "en_US": {
+                                      "type": ["null", "string"]
+                                    },
+                                    "fr_FR": {
+                                      "type": ["null", "string"]
+                                    }
+                                  }
+                                }
+                              }
+                            },
+                            "label": {
+                              "type": ["null", "string"]
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              },
+              "responseEditable": {
+                "type": ["null", "boolean"]
+              },
+              "responseRequired": {
+                "type": ["null", "boolean"]
+              }
+            }
+          }
+        },
+        "description": {
+          "type": ["null", "object"],
+          "properties": {
+            "localized": {
+              "type": ["null", "object"],
+              "properties": {
+                "da_DK": {
+                  "type": ["null", "string"]
+                },
+                "de_DE": {
+                  "type": ["null", "string"]
+                },
+                "en_US": {
+                  "type": ["null", "string"]
+                },
+                "fr_FR": {
+                  "type": ["null", "string"]
+                }
+              }
+            }
+          }
+        },
+        "postSubmissionInfo": {
+          "type": ["null", "object"],
+          "properties": {
+            "message": {
+              "type": ["null", "object"],
+              "properties": {
+                "localized": {
+                  "type": ["null", "object"],
+                  "properties": {
+                    "da_DK": {
+                      "type": ["null", "string"]
+                    },
+                    "de_DE": {
+                      "type": ["null", "string"]
+                    },
+                    "en_US": {
+                      "type": ["null", "string"]
+                    },
+                    "fr_FR": {
+                      "type": ["null", "string"]
+                    }
+                  }
+                }
+              }
+            },
+            "callToAction": {
+              "type": ["null", "object"],
+              "properties": {
+                "callToActionLabel": {
+                  "type": ["null", "string"]
+                },
+                "callToActionTarget": {
+                  "type": ["null", "object"],
+                  "properties": {
+                    "landingPageUrl": {
+                      "type": ["null", "string"]
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "created": {
```

Comment:
It has created field that looks a lot like a timestamp. Can we use this for incremental sync?

Reply:
@natikgadzhi we can not, according to the official documentation we can only use `submittedAt` filtering, and the issue is, as explained, on the way to build the API call `submittedAtTimeRange=(start:1711407600000,end:1711494000000)` which does not look feasible with low code
https://learn.microsoft.com/en-us/linkedin/marketing/lead-sync/leadsync?view=li-lms-2024-06&tabs=http#get-multiple-lead-form-responses

if you have any workaround suggestion 🙏